clustering/jgroups/extension/src/main/resources/subsystem-templates/jgroups.xml

```diff
@@ -7,9 +7,9 @@
             <channel name="ee" stack="udp" cluster="ejb"/>
         </channels>
         <stacks>
-            <stack name="tcp">
```

Comment:
Note that the diff in this file looks complex but really all that's done is moving the entire xml element with stack name="tcp" below the one with name="udp". Since most of the child elements in each stack are the same you end up with a complex looking diff for a simple change.